gui/default/assets/css/tree.css

```
@@ -56,3 +56,91 @@
     white-space: normal;
     word-break: break-all;
 }
+
+.fancytree-temp .fancytree-title {
```

```suggestion
.folderTree-new-folder {
```

You can apply this to `tr` directly.